developer-tooling/scripts/coverage_runner.py

```
@@ -0,0 +1,85 @@
+        env['INVENTREE_SITE_URL'] = 'http://localhost:8000'
+        env['INVENTREE_BACKUP_DIR'] = backup_dir
+
+        # 1. Run migrations to verify they work and to initialize the database
+        print('Verifying and applying squashed database migrations...')
+        migrate_cmd = [
+            sys.executable,
+            'src/backend/InvenTree/manage.py',
+            'migrate',
+            '--noinput',
+        ]
+        migrate_res = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)